### ISSUE TITLE
Replace checkbox disabled for a command in an MNX

### ISSUE THREAD

**Issue by Jimrnelson** [VFPX/GoFish#133]
See the screenshot below.

I have matches in the Command for an MNX, and the checkboxes to allow replacement are not visible.

![image](https://github.com/VFPX/GoFish/assets/81179335/2009ed57-2f01-4d90-a255-b159ea1fbc35)

**Comment by DougHennig** [comment]
I believe Lutz determined that replacement in an MNX was too dangerous so disabled that ability.

**Comment by lscheffler** [comment]
For all I remember, the replacement for mnx should work on the higher replacement levels. I only blocked mpr out.

**Comment by Jimrnelson** [comment]
@Jimrnelson 

> For all I remember, the replacement for mnx should work on the higher replacement levels. I only blocked mpr out.

That may have been your intent, but those checkboxes are not appearing.

**Comment by myearwood1** [comment]
I co-founded the Toronto FoxPro users's group. I invited Drew Speedie to speak. I then arranged a class where Drew taught for a week. Drew decided to hire me and they paid my college tuition to get into the US. Several in this community treated my like garbage, including your over-highly vaunted Lutz. I offer my services as a far more skillful developer on the grounds that ANYONE, who does anything illegal as Lutz's LIBEL against me, is banned immediately.
